app/main/lib/similarity.py

```diff
@@ -133,7 +133,7 @@ def callback_add_item(item, similarity_type):
     if function:
         response = function(item)
         app.logger.info(f"[Alegre Similarity] CallbackAddItem: [Item {item}, Similarity type: {similarity_type}] Response looks like {response}")
```

Comment:
drive-by comment: please also update the `Response looks like {response}` bit to reflect what's newly returned for more useful :wood:s thank u